cmd/dfns_example/main.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github/dfns/dfns-sdk-go/pkg/credentials"
+	dfnsApi "github/dfns/dfns-sdk-go/pkg/dfns-api-client"
+)
+
+func main() {
+	// Load environment variables from .env file
+	if _, err := os.Stat(".env"); os.IsNotExist(err) {
+		log.Println("The .env file does not exist.")
+	} else {
+		// Load environment variables from .env file
+		if err := godotenv.Load(".env"); err != nil {
+			log.Fatal("Error loading .env file:", err)
+		}
+	}
+
+	dfnsConfig := LoadConfig()
+	apiConfig := dfnsConfig.GetDfnsApiConfig()
+	keySignerConfig, err := dfnsConfig.GetKeySignerConfig()
+	if err != nil {
+		log.Fatalln("Error getting KeySignerConfig:", err)
+	}
+
+	signer := credentials.NewAsymmetricKeySigner(keySignerConfig)
+
+	apiOptions, err := dfnsApi.NewDfnsBaseApiOptions(
+		apiConfig,
+		signer,
+	)
+	if err != nil {
+		log.Fatalln("Error creating DfnsBaseApiOption:", err)
+	}
+
+	httpClient := dfnsApi.CreateDfnsApiClient(apiOptions)
+
+	// Retrieve a wallet
+	if err := getWallet(httpClient, apiOptions); err != nil {
+		log.Fatalln("Error fetching wallet:", err)
+	}
+
+	// Create a wallet
+	if err := createWallet(httpClient, apiOptions); err != nil {
+		log.Fatalln("Error creating wallet:", err)
+	}
```

Comment:
maybe reverse these two calls? might be confusing to have a hardcoded wallet they can't access. would be more intuitive to do 

* first create wallet
* then get wallet balance using the above wallet id.